docs/source/guide/ml.md

```diff
@@ -1,144 +1,163 @@
 ---
-title: Machine learning backend
+title: Set up machine learning with your labeling process
 type: guide
-order: 906
+order: 606
 ---
 
-You can easily connect your favorite machine learning framework with Label Studio Machine Learning SDK. 
+Set up machine learning with your labeling process by setting up a machine learning backend for Label Studio. 
 
-That gives you the opportunities to use:
-- **Pre-labeling**: Use model predictions for pre-labeling (e.g. make use of on-the-fly model predictions for creating rough image segmentations for further manual refinements)
-- **Autolabeling**: Create automatic annotations
-- **Online Learning**: Simultaneously update (retrain) your model while new annotations are coming
-- **Active Learning**: Perform labeling in active learning mode - select examples model is uncertain about
-- **Prediction Service**: Instantly create running production-ready prediction service
+With Label Studio, you can set up your favorite machine learning models to do the following:
+- **Pre-labeling** by letting models predict labels and then perform further manual refinements. 
+- **Auto-labeling** by letting models create automatic annotations. 
+- **Online Learning** by simultaneously updating your model while new annotations are created, letting you retrain your model on-the-fly. 
+- **Active Learning** by selecting example tasks that the model is uncertain how to label for your annotators to label. 
 
-
-## Tutorials
-
-- [Create the simplest ML backend](/tutorials/dummy_model.html)
-- [Text classification with Scikit-Learn](/tutorials/sklearn-text-classifier.html)
-- [Transfer learning for images with PyTorch](/tutorials/pytorch-image-transfer-learning.html)
-- [Image Object Detector](/tutorials/object-detector.html)
-- [Chatbot response generation with HuggingFace's GPT2 model](/tutorials/gpt.html)
-- [Automatic Speech Recognition with Nvidia's NeMo](/tutorials/nemo_asr.html)
-
-#### Create ML backend
-
-Check examples in [`label-studio/ml/examples`](https://github.com/heartexlabs/label-studio/tree/master/label_studio/ml/examples) directory.
+With these capabilities, you can use Label Studio as part of a production-ready **Prediction Service**. 
 
 ## Quickstart
 
-Here is a quick example tutorial on how to run the ML backend with a simple text classifier:
+Get started with a machine learning (ML) backend with Label Studio. You need to start both the machine learning backend and Label Studio to start labeling. You can review examples in the [`label-studio/ml/examples` section of the repository](https://github.com/heartexlabs/label-studio/tree/master/label_studio/ml/examples).
```

Comment:
ML backend has been moved to https://github.com/heartexlabs/label-studio-ml-backend